src/System.Threading/tests/ReaderWriterLockSlimTests.cs

```diff
@@ -383,6 +383,7 @@ public static void WriterToUpgradeableReaderChain()
 
         [Fact]
         [OuterLoop]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17443")] // Hangs in desktop
```

Comment:
Yes that's right, I wasn't aware of this attribute at the time. Thanks for fixing, the original issue was https://github.com/dotnet/corefx/issues/3364. The plan is to eventually port this fix to desktop as well.